### PR TITLE
Add support for being launched is service (#8)

### LINKED ISSUE #8: Help running as a service on ubuntu
so i am trying to setup this a service on ubuntu but get this error > 2020-04-11 08:40:59.192088181 +0000 UTC m=+0.006642289: An error occurred when attempting to read the input to pass as input to the console
Details: EOF

from my services file >
[Unit]
Description=Factorio1 Server service
After=network.target

[Service]
User=noidea
WorkingDirectory=/home/noidea/FactoCord-1
ExecStart=/home/noidea/FactoCord-1/FactoCord-1
Restart=always

[Install]
WantedBy=multi-user.target

### PR DESCRIPTION
**Pull request by TechnoStrife**
Solves #8
Needs testing

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -81,15 +81,20 @@ func main() {
...
 	go func() {
 		Console := bufio.NewReader(os.Stdin)
+		service := support.Config.PossiblyService
 		for {
 			line, _, err := Console.ReadLine()
 			if err != nil {
+				if service {
+					return
+				}
 				support.ErrorLog(fmt.Errorf("%s: An error occurred when attempting to read the input to pass as input to the console\nDetails: %s", time.Now(), err))
```
Comment (maxsupermanhd) [inline main.go:91]: Maybe move that before service check to still have log entry?

Comment (TechnoStrife) [reply main.go:91]: Notice that support.ErrorLog kills the whole application if called. So we need a new function that will only put the error into the file. 

But do we need that if the user specified that console will probably be unavailable?

Comment (maxsupermanhd) [reply main.go:91]: With some effort in service file (like I did) you can run service with tmux attached console so no point here. You can also make another flag for muting console output :shrug:

**Comment by maxsupermanhd** [comment]
Closing as author of issue still potentially afk and this is not need. (Works fine on my system)

**Pull request closed without merging**